feature-libs/product/future-stock/components/future-stock-trigger/future-stock-trigger.component.ts

```diff
@@ -1,13 +1,44 @@
 import {
   ChangeDetectionStrategy,
   Component,
+  OnInit,
 } from '@angular/core';
-
+import { FutureStock } from '../../root/models/future-stock.model';
```

Comment:
If you do imports from different entry points I recommend referencing the specific package: 
`@spartacus/product/future-stock/root`

Reply:
done